tensorflow/contrib/cmake/README.md

```diff
@@ -13,23 +13,59 @@ Linux.
 Current Status
 --------------
 
+TensorFlow release 0.12 officially supports Windows, with a native TensorFlow package 
+provided for Windows 7, 10 and Server 2016.
+The supported languages include Python 3.5 64-bit (from Python.org or Anaconda) and C++. 
+This release enables acceleration with CUDA 8.0 and cuDNN 5.1 compatible GPUs.
+A [pip package](https://pypi.python.org/pypi/tensorflow) was published and the
+installation can be done with a single command:
+
+`C:\> pip install tensorflow`
+
+And for GPU support:
+
+`C:\> pip install tensorflow-gpu`
```

Comment:
Absolutely! Much conciser. I ended up placing these notes because it was right after the pre-release and I am not sure if there was an updated official doc already on tensorflow.org. 
Will replace as you've said.